hw2/04-api.html

```diff
@@ -14,6 +14,43 @@
     <section class="container border rounded bg-light w-50 mx-auto mt-5 p-3">
       <h1 class="mt-2 mb-4">REST Countries</h1>
       <ol id="results"></ol>
+      
     </section>
   </body>
+
+  <script>
```

Comment:
JavaScript would be better in it's own file for readability.  
Items need to be in order.
The population's need to be formatted correctly.
Code looks good.